pkg/kube/client.go

```diff
@@ -55,6 +58,10 @@ func New(getter genericclioptions.RESTClientGetter) *Client {
 	if getter == nil {
 		getter = genericclioptions.NewConfigFlags(true)
 	}
+	if err := apiextv1beta1.AddToScheme(scheme.Scheme); err != nil {
```

Comment:
We should add a comment here so people don't have to do the same research you did